### PR TITLE
Populate CWE finding field from CycloneDX JSON file

### PR DESCRIPTION
**Pull request by coheigea**

### CODE REVIEW

File: dojo/tools/cyclonedx/parser.py [hunk 1]
```diff
@@ -349,6 +349,14 @@ def _get_findings_json(self, file, test):
                 if vulnerability_ids:
                     finding.unsaved_vulnerability_ids = vulnerability_ids
 
+                # if there is some CWE
+                cwes = vulnerability.get('cwes')
+                if cwes and len(cwes) > 1:
+                    # FIXME support more than one CWE
+                    LOGGER.debug(f"more than one CWE for a finding {cwes}. NOT supported by parser API")
```
Comment (StefanFl) [inline dojo/tools/cyclonedx/parser.py:356]: Why don't we take just the first CWE? Better than none.

**Review by StefanFl — APPROVED** [review]

**Review by StefanFl — CHANGES_REQUESTED** [review]
see comment

File: dojo/tools/cyclonedx/parser.py (hunk 1 above)
Comment (coheigea) [reply dojo/tools/cyclonedx/parser.py:356]: @StefanFl It does actually take the first CWE in all cases. This if statement just logs a warning about there being more than one CWE, the next if statement takes the first CWE.

**Comment by coheigea** [comment]
@StefanFl Please review again

**Review by StefanFl — APPROVED** [review]

**Review by damiencarol — APPROVED** [review]

**Pull request merged**